odl/solvers/functional/functional.py

```diff
@@ -822,6 +823,83 @@ def __str__(self):
                                           self.translation)
 
 
+class InfimalConvolution(Functional):
+
+    """The functional representing ``h(x) = inf_y f(x-y) + g(y)``."""
```

Comment:
One huge pull request fixing documentation would be something like christmas eve for us ;)